cypress/e2e/36-versions.cy.ts

```diff
@@ -0,0 +1,38 @@
+import { INSTANCE_OWNER } from '../constants';
+import { WorkflowsPage } from '../pages/workflows';
+import {
+	closeVersionUpdatesPanel,
+	getVersionCard,
+	openVersionUpdatesPanel,
+} from '../composables/versions';
+
+const workflowsPage = new WorkflowsPage();
+
+describe('Versions', () => {
+	it('should open updates panel', () => {
+		cy.intercept('GET', '/rest/settings', (req) => {
+			req.continue((res) => {
+				if (res.body.hasOwnProperty('data')) {
+					res.body.data = {
+						...res.body.data,
+						releaseChannel: 'stable',
+						versionCli: '1.0.0',
+						versionNotifications: {
+							enabled: true,
+							endpoint: 'https://api.n8n.io/api/versions/',
```

Comment:
should not we mock the response from this endpoint?

Reply:
Good point! Done.